androidApp/src/debug/res/xml/shortcuts.xml

```diff
@@ -0,0 +1,15 @@
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+  <shortcut
+    android:shortcutId="add_new_note"
+    android:enabled="true"
+    android:icon="@drawable/ic_launcher_shortcut"
+    android:shortcutShortLabel="@string/shortcut_short_label"
+    android:shortcutLongLabel="@string/shortcut_long_label"
```

Comment:
A long label is only needed if it's different from the short label. Let's remove this.

> An extended phrase that describes the shortcut's purpose. If there's enough space, the launcher displays this value instead of android:shortcutShortLabel. 

https://developer.android.com/guide/topics/ui/shortcuts/creating-shortcuts#static